### DISCUSSION TITLE
有办法获取canvas 滚动宽度吗，出现滚动条的时候，我想获取真实的滚动宽度

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [antvis/S2#1765] [Q&A] [answered]
有办法获取canvas 滚动宽度吗，出现滚动条的时候，我想获取真实的滚动宽度

现在遇到个需求：

   想定位一个元素到canvas上，上下左右滚动的时候，定位的元素可以跟着单元格走而不是固定在一个地方。

**Comment by YardWill** [comment, marked as answer]
可以监听S2Event.GLOBAL_SCROLL事件，滚动后会触发然后抛出当前scrollX和scrollY
